pkg/services/alerting/eval_context.go

```diff
@@ -124,7 +124,14 @@ func (c *EvalContext) GetRuleURL() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return fmt.Sprintf(urlFormat, models.GetFullDashboardUrl(ref.Uid, ref.Slug), c.Rule.PanelID, c.Rule.OrgID), nil
+
+	u := fmt.Sprintf(urlFormat, models.GetFullDashboardUrl(ref.Uid, ref.Slug), c.Rule.PanelID, c.Rule.OrgID)
+	if c.Rule != nil {
+		from := c.Rule.LastStateChange
+		to := from.Add(c.Rule.For)
+		u += fmt.Sprintf("&from=%d&to=%d", from.UnixNano()/int64(time.Millisecond), to.UnixNano()/int64(time.Millisecond))
```

Comment:
I wonder if it would make sense to put some kind of time padding around so that the relevant events are not right on the edges of the graph. cc @marefr

Reply:
I added a padding of 5 minutes to the dashboard link.